### ISSUE TITLE
Inconsistent language code: Esperanto flag uses full name instead of ISO 639-1 code ‘eo’

### ISSUE THREAD

**Issue by BaksiLi** [monero-project/monero-gui#4383]
The language flag for Esperanto (`lang/flags/esperanto.png`) is inconsistently named compared to other language flags in the same directory.

Current state:
- File is named `esperanto.png` (full language name)
- Other language flags use ISO 639-1 two-letter codes (e.g., `gb.png`, `fr.png`)

Expected state:
- File should be named `eo.png` (using ISO 639-1 code for Esperanto)

Reference:
- ISO 639-1 code for Esperanto is "eo"
- [ISO 639-1 Language Codes Reference](https://www.loc.gov/standards/iso639-2/php/English_list.php)

**Comment by BaksiLi** [comment]
BTW, https://translate.getmonero.org/engage/monero/ seems down, Error 502

**Comment by jermanuts** [comment]
Yes, this issue have been for years, no one stepped up to fix Weblate yet.

**Comment by selsta** [comment]
Resolved in #4384.
